jpeg-compare.c

```diff
@@ -96,13 +97,15 @@ int compareFastFromBuffer(unsigned char *imageBuf1, long bufSize1, unsigned char
 
     // Cleanup
     free(hash1);
+    hash1 = NULL;
```

Comment:
Maybe?:
```c
*hash1 = NULL;
*hash2 = NULL;
```

Reply:
You cannot modify memory after it has been free()'d.

Reply:
Sorry. Blunder.